developer-docs-site/docs/tutorials/first-transaction.md

```diff
@@ -181,9 +222,9 @@ The following demonstrate how to read data from the blockchain and how to write
   </TabItem>
 </Tabs>
 
-## Step 3) Faucet interface
+## Step 3: Faucet interface
 
-Blockchain faucets issue test tokens to accounts. These can be used for testing, e.g., paying gas fees or transferring tokens between users. The Aptos faucet can also create accounts if they do not exist yet. The Aptos faucet interface requires a public key represented in a hex-encoded string.
+Aptos Blockchain faucets issue test tokens to accounts. These test tokens can be used for testing, e.g., paying gas fees or transferring tokens between users. The Aptos Faucet can also create accounts if they do not exist. The Aptos Faucet interface requires a public key represented in a hex-encoded string.
```

Comment:
What do you think about the terminologies here: coins vs tokens? I think we've been using them differently (coins = tokens, tokens = NFTs) from the rest of the crypto world (coin = native coin on a network, token = ERC-20 style interfaces)

Reply:
I have it in my to-do list to address this. Totally agree. I want to address it at an overall higher level and do the change in all our docs.